mkdocs/assets/search/mkdocs/js/tipuesearch_content.js

```diff
@@ -0,0 +1,4 @@
+var tipuesearch = {{ search_index }};
+for (var i=0; i < tipuesearch.pages.length; i++){
+    tipuesearch.pages[i].loc = base_url + tipuesearch.pages[i].loc;
+}
```

Comment:
I can't think why we would want to do this? Just to make it easier to upgrade and potentially ship multiple outputs?